src/cookie.js

```diff
@@ -51,5 +51,5 @@ export class Cookie {
 
 // support node-requires (for test import)
 if (typeof module !== 'undefined' && module.exports) {
-    module.exports = Cookie
```

Comment:
this is to match how this class is exported at the top:

> export class Cookie

makes sure that export is the same in browser vs node

Reply:
Can we just make the test es6 modules compliant so we can remove this instead?

Reply:
Yeah, good point. I ended up removing and… everything still seems to work? Both here and in extension tests pass.